consensus/src/block_storage/sync_manager.rs

```diff
@@ -188,7 +188,7 @@ impl BlockStore {
         if highest_commit_cert.ledger_info().ledger_info().ends_epoch() {
             retriever
                 .network
-                .notify_epoch_change(EpochChangeProof::new(
+                .send_epoch_change(EpochChangeProof::new(
```

Comment:
I was trying to differentiate it from send but maybe did a bad job. notify == notify self so this doesn't go through network